spec/broadside/ecs/ecs_manager_spec.rb

```diff
@@ -9,28 +9,38 @@
   let(:cluster) { 'cluster' }
   let(:name) { 'job' }
 
-  it 'create_service' do
-    expect { described_class.create_service(cluster, service_name) }.to_not raise_error
+  describe '#create_service' do
```

Comment:
again i don't really see much advantage to having a whole `describe` block - just adjust the `it` message if you want more clarity
```ruby
it `#create_service creates an ECS service from the given configs' do
  ...
end
```

in general i stick to only defining a new context when it is necessary for scoping variables for a set of specs.